modules/popup/react/lib/useCloseOnEscape.ts

```diff
@@ -6,23 +6,33 @@ import {PopupStack} from '@workday/canvas-kit-popup-stack';
  * Registers global detection of the Escape key. It will only call the `onClose` callback if the
  * provided `stackRef` element is the topmost in the stack. The `stackRef` should be the same as the one passed
  * to `usePopupStack` or the `Popper` component since `Popper` uses `usePopupStack` internally.
+ * If provided, the `previousFocusRef` allows you to return focus to an element before `onClose` is called
  * @param stackRef
  * @param onClose
+ * @param previousFocusRef
```

Comment:
I'm not sure about the name `previousFocusRef` as it could be any element, but that's my intended use case.

Reply:
What about `returnFocusRef`?

Reply:
Thanks! Updated to use `returnFocusRef` 👍